### DISCUSSION TITLE
Doris支持多表写入吗

### DISCUSSION THREAD

**Discussion by SamuelBin** [apache/doris#13043] [A - General / Q&A]
现在将stream里的数据写到多个表里，都是用doris sink，能不能使用一个doris sink就可以

**Comment by hf200012** [comment]
不支持

**Comment by hf200012** [comment]
你可以参考这个：https://github.com/apache/doris/tree/master/samples/doris-demo/flink-demo-v1.1
实现flink cdc 整库同步

**Reply by SamuelBin** [reply]
是这样，我们是要将kafka的数据，根据不同的企业，写入到不同的以企业id为库的事件表里，但是目前遇到的就是需要对每个库表都要提供个doris sink，有新的企业创建上报数据，就得手动添加一个sink，很麻烦
